packages/legend-application-repl/src/components/dataCube/grid/DataCubeGrid.tsx

```diff
@@ -61,6 +64,8 @@ console.error = (message?: unknown, ...agrs: unknown[]): void => {
   console.log(`%c ${message}`, 'color: silver'); // eslint-disable-line no-console
 };
 
+ModuleRegistry.registerModules([CsvExportModule, ExcelExportModule]);
```

Comment:
let's not do it here, we register new modules as part of component declaration, see line 334

Reply:
done